modules/about-bare-metal-hosts-and-nodes.adoc

```diff
@@ -0,0 +1,12 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/managing-bare-metal-hosts.adoc
+
+[id="about-bare-metal-hosts-and-nodes_{context}"]
+= About bare metal hosts and nodes
+
+In order to provision a {op-system-first} bare metal host as a node in your cluster, you must first create a `MachineSet` CR object that corresponds to the bare metal host hardware. Bare metal host machine sets describe infrastructure components specific to your configuration. You apply specific Kubernetes labels to these machine sets and then update the infrastructure components to run on only those machines.
+
+When a `Machine` CRD has been created for your hardware host, then create a `Machine` CR object. `Machine` CR's are used by {product-title} to provision the bare metal node that corresponds to the host as specified in the `MachineSet` CR. Bare metal nodes are created automatically from the relevant `Machine` CR.
```

Comment:
1. sp out CRD first.
2. passive voice. What creates the Machine CRD? After x creates the Machine CRD for you hardware host, create a Machine CR object.
3. {product-title} uses Machine CRs to provision.. 
4. The relevant Machine CR creates the bare metal nodes automatically.